.github/workflows/openbsd.yml

```diff
@@ -0,0 +1,48 @@
+name: C CI on OpenBSD
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: OpenBSD-${{ matrix.config.name }}
+    runs-on: macos-12
```

Comment:
`macos-12` is accurate, but confusing. What about a comment above this line? Eventually a link to https://github.com/marketplace/actions/openbsd-vm

Reply:
Done.